File: Day03/ex16.R
Produit <- c("Pomme", "Orange", "Banane", "Raisin")
Quantite <- c(50, 30, 20, 15)
PrixUnitaire <- c(1.2, 0.8, 0.5, 2.0)

ventes <- data.frame(Produit, Quantite, PrixUnitaire)

ventes$Total <- c(ventes$PrixUnitaire * ventes$Quantite)

print(ventes$Total)

prix_total <- 0
for (i in ventes$Total) {
  prix_total = prix_total + i
}

print(prix_total)

File: Day04/ex16.js
let votes = ["Pomme", "Banane", "Orange", "Pomme", "Raisin", "Banane", "Banane"];
let resultats = {};

for (let i = 0; i < votes.length; i++) {
    let fruit = votes[i];
    if (resultats[fruit]) {
        resultats[fruit]++;
    } else {
        resultats[fruit] = 1;
    }
}

console.log(resultats);

let fruitGagnant = "";
let maxVotes = 0;
for (let fruit in resultats) {
    if (resultats[fruit] > maxVotes) {
        maxVotes = resultats[fruit];
        fruitGagnant = fruit;
    }
}
console.log(fruitGagnant);
